Makefile

```diff
@@ -221,6 +230,16 @@ auraed-start: ## Starts the installed auraed executable
 
 # Commands for other crates
 
+.PHONY: ebpf-build ## Build auraed eBPF probes (debug)
+ebpf-build: nightly bpf-linker
+	cd ./aurae-ebpf && \
+		$(cargo) +nightly build --target=bpfel-unknown-none -Z build-std=core
+
+.PHONY: ebpf-release ## Build auraed eBPF probes
+ebpf-release: nightly bpf-linker
+	cd ./aurae-ebpf && \
+		$(cargo) +nightly build --package aurae-ebpf --target=bpfel-unknown-none -Z build-std=core --release
```

Comment:
is this different to `$(cargo) +nightly build --path ./aurae-ebpf --package ...` ?

Reply:
I'm not showing `--path` as a valid `cargo build` flag. I'm on cargo 1.66.1.

```error: unexpected argument '--path' found```

Reply:
ah looks like we use install for other executables.